components/Header/NavBar.tsx

```diff
@@ -3,10 +3,12 @@ import React from 'react';
 import AuthButton from './AuthButton';
 import HomeButton from './HomeButton';
 import Help from './Help';
+import UserAccountButton from './UserAccountButton';
 
 const NavBar: React.FC = () => (
   <nav className='w-full flex-none flex justify-center border-b shadow-md bg-primary/10 shadow-primary/10 h-16'>
     <div className='w-full max-w-[614px] md:max-w-[80%] flex items-center p-3 text-sm'>
+      <UserAccountButton />
```

Comment:
Have a look at [this component](https://github.com/CM20254-Group-7/rental-review/blob/e3985fb9f3f777254414d43a5b1c5494a920e3a2/components/Header/NavBar.tsx#L18), as I think they will want merging

Reply:
Can you elaborate on this as I don't quite understand what you mean

Reply:
It looks like it's been removed? but there was also a link to the account page in the Auth dropdown, so I was suggesting that you probably wanted to combine them to prevent duplication.

Reply:
Figured it out, PR #65 decided to replace the account link with a dashboard link rather than adding one

Reply:
I think that's still the right place for that link in the header

![image](https://github.com/CM20254-Group-7/rental-review/assets/74842299/c5f04b78-2dd9-4060-b86d-09af1c19e9bd)

Move this link to replace the dashboard in the dropdown, as we already have a dashboard link in the main header

Reply:
Ah yeah I removed it and Could not figure out why the button wasn't working